package-lock.json

```diff
@@ -1,8 +1,4849 @@
 {
   "name": "harmony-marketplace-sdk",
   "version": "1.0.0",
-  "lockfileVersion": 1,
```

Comment:
why is there a package lock diff here? I don't see any installs

Reply:
Oh my, that was because I deleted the `package-lock` and `node_modules` file before install the new dependencies, I'll change it right away!